local-volume/provisioner/pkg/common/common.go

```diff
@@ -122,13 +126,25 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 	}
 }
 
-// GetVolumeConfigFromConfigMap gets volume configuration from given configmap,
-func GetVolumeConfigFromConfigMap(client *kubernetes.Clientset, namespace, name string) (map[string]MountConfig, error) {
-	configMap, err := client.CoreV1().ConfigMaps(namespace).Get(name, metav1.GetOptions{})
+// GetVolumeConfigFromConfigMap gets volume configuration from the mounted configmap volume,
+func GetVolumeConfigFromConfigMap() (map[string]MountConfig, error) {
+	volumes := make(map[string]string)
+	files, err := ioutil.ReadDir(provisionerConfigPath)
 	if err != nil {
 		return nil, err
 	}
-	return ConfigMapDataToVolumeConfig(configMap.Data)
+	for _, file := range files {
+		if !file.IsDir() {
+			b, err := ioutil.ReadFile(path.Join(provisionerConfigPath, file.Name()))
+			fmt.Printf("><><><> reading file: %s <><><><\n", path.Join(provisionerConfigPath, file.Name()))
+			if err != nil {
+				return nil, err
```

Comment:
We should probably log an error about the specific file, aka storageClassName that failed to be read.